src/main/java/teammates/common/util/HttpRequestHelper.java

```diff
@@ -36,9 +36,9 @@ public static String getValueFromParamMap(Map<String, String[]> paramMap, String
     }
 
     /**
-     * Returns a HashMap object containing all the parameters key-value pairs from a URLFetchRequest object.
+     * Returns a Map interface containing all the parameters key-value pairs from a URLFetchRequest object.
```

Comment:
No need interface. Here
`Returns a Map containing all the parameters key-value .....`